<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="CDisplay" name="display">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="CPlayField" name="playField">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Spiel</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSize">
    <property name="title">
     <string>Spielfeld</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBombs">
    <property name="title">
     <string>Bomben</string>
    </property>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuSize"/>
   <addaction name="menuBombs"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPlayField</class>
   <extends>QFrame</extends>
   <header location="global">cplayfield.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CDisplay</class>
   <extends>QLabel</extends>
   <header>cdisplay.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
